### DISCUSSION TITLE
Could really use some help with background colors in HTML and CSS

### DISCUSSION THREAD

**Discussion by DanielGrayPet** [community/community#21408] [Other Feature Feedback, Questions, & Ideas]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/a/7/a7c26d38a4a04124d6dd3ce3fc3f243770c3e7eb.jpeg" data-download-href="/uploads/short-url/nW4igWSly13l3ObI9pqprijknVN.jpeg?dl=1" title="check.PNG" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/44482255/181064444-1174ea98-d001-4fa0-b69c-7b508e995dea.jpeg" alt="check.PNG" data-base62-sha1="nW4igWSly13l3ObI9pqprijknVN" width="690" height="368" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/a/7/a7c26d38a4a04124d6dd3ce3fc3f243770c3e7eb_2_690x368.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/a/7/a7c26d38a4a04124d6dd3ce3fc3f243770c3e7eb_2_1035x552.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/a/7/a7c26d38a4a04124d6dd3ce3fc3f243770c3e7eb_2_1380x736.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/a/7/a7c26d38a4a04124d6dd3ce3fc3f243770c3e7eb_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">check.PNG</span><span class="informations">1892×1010 148 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>Hi,</p>
<p>I have trouble understanding why I have such big white strips on the sides of my page (Pic attached). I’ve tried looking for a fix a while now but nothing seems to really work. Couldn’t find anything in my code that might cause this too.</p>
<p>Appreciate any help.</p>

**Comment by jfullstackdev** [comment]
<p>if that is a public project, you can send the direct link, it’s best, so we can take a closer look</p>

**Comment by DanielGrayPet** [comment]
<p>It’s not a public repository. Any other way I could send relevant snippets of the code?</p>

**Comment by jfullstackdev** [comment]
<p>screenshots will do and pasting of code here</p>

**Comment by DanielGrayPet** [comment]
<p>index.js ← the page that is in the screenshot</p>
<pre><code class="lang-auto">import * as React from "react"
import Layout from "../components/Layout"
import { container, heading1, body1, heading2, picturelist, picturedo, html, body } from "../components/index.module.css"
import { StaticImage } from "gatsby-plugin-image"

const HomePage = () =&gt; {
  return (
    &lt;&gt;
    &lt;html className={html}&gt;
      &lt;Layout pageTitle="Homepage"&gt;&lt;/Layout&gt;
      &lt;body className={body}&gt;
      &lt;div className={container}&gt;
        &lt;h1 className={heading1}&gt;What We Do&lt;/h1&gt;
        &lt;body&gt;
          &lt;p className={body1}&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean mollis mi ac tellus rhoncus, sit amet efficitur lorem viverra. Nam augue justo, posuere eu mi ut, aliquam auctor enim. Praesent nisi tortor, facilisis vestibulum massa eu, lobortis cursus metus. Ut iaculis pulvinar rhoncus. Praesent consectetur felis a orci aliquet, faucibus eleifend enim rhoncus. Curabitur vitae tellus eu ligula auctor facilisis. Nulla venenatis vehicula tortor, sed hendrerit enim maximus sed. Sed id malesuada odio, dapibus finibus ligula. Fusce rhoncus, eros sit amet feugiat maximus, dui nisi commodo eros, eu ultricies leo nulla eget metus. Nunc vel nisl at diam dapibus ultrices et nec sem. Nulla facilisi.&lt;/p&gt;
        &lt;/body&gt;
      &lt;/div&gt;
      &lt;div className={container}&gt;
        &lt;h2 className={heading2}&gt;Sneak Peek&lt;/h2&gt;
        &lt;body&gt;
          &lt;ul className={picturelist}&gt;
            &lt;li className={picturedo}&gt;&lt;StaticImage src= "../images/image4.jpg" alt="silly" placeholder="blurred"&gt;&lt;/StaticImage&gt;&lt;/li&gt;
            &lt;li className={picturedo}&gt;&lt;StaticImage src= "../images/image6.jpg" alt="silly" placeholder="blurred"&gt;&lt;/StaticImage&gt;&lt;/li&gt;
            &lt;li className={picturedo}&gt;&lt;StaticImage src= "../images/image5.jpg" alt="silly" placeholder="blurred"&gt;&lt;/StaticImage&gt;&lt;/li&gt;
          &lt;/ul&gt;
        &lt;/body&gt;
      &lt;/div&gt;
      &lt;/body&gt;
      &lt;/html&gt;
    &lt;/&gt;
  );
}

export default HomePage
</code></pre>
<p>index.module.css ← the css module for the index page</p>
<pre><code class="lang-auto">.container{
    background: ghostwhite;
    font-size: 20px;
    padding: 10px;
    border: 1px solid lightgray;
    margin: 10px;
}
.heading1{
    color: rgb(0, 34, 102);
    text-align: center;
    font-size: 48px;
}
.body1{
    color: darkblue;
    text-align: center;
    font-size: 24px;
}
.heading2{
    color: rgb(0, 34, 102);
    text-align: center;
    font-size: 48px;
}
.picturelist{
    align-items: center;
    justify-content: center;
    display: flex;
    list-style: none;
    padding-left: 0;
}
.picturedo{
    height: auto;
    width: auto;
    padding: 60px 120px;
}
.html{
    background: #11001A;
    background: -moz-linear-gradient(45deg, #11001A 0%, #00004D 70%, #000000 100%);
    background: -webkit-linear-gradient(45deg, #11001A 0%, #00004D 70%, #000000 100%);
    background: linear-gradient(45deg, #11001A 0%, #00004D 70%, #000000 100%);
}
.body{
    min-height: 100%;
    min-width: 100%;
    margin: 0px;
}
</code></pre>
<p>global.css</p>
<pre><code class="lang-auto">html {
    background: #11001A;
    background: -moz-linear-gradient(45deg, #11001A 0%, #00004D 70%, #000000 100%);
    background: -webkit-linear-gradient(45deg, #11001A 0%, #00004D 70%, #000000 100%);
    background: linear-gradient(45deg, #11001A 0%, #00004D 70%, #000000 100%);
}
</code></pre>
<p>gatsby-browser.js</p>
<pre><code class="lang-auto">const React = require("react")
const Layout = require("./src/components/layout")

// Logs when the client route changes
exports.onRouteUpdate = ({ location, prevLocation }) =&gt; {
  console.log("new pathname", location.pathname)
  console.log("old pathname", prevLocation ? prevLocation.pathname : null)
}

// Wraps every page in a component
exports.wrapPageElement = ({ element, props }) =&gt; {
  return &lt;Layout {...props}&gt;{element}&lt;/Layout&gt;
}

//import stylesheet
import "./src/styles/global.css"
</code></pre>

**Comment by DanielGrayPet** [comment, marked as answer]
<p>Found the solution to my problem. Thanks for anyone trying to help.<br>
I created a global.css stylesheet and implemented it correctly this time to gatsby-browser.jsx.<br>
You need to restart the development server after creating gatsby-browser.jsx so that’s why it didn’t show yet.<br>
Thank you all.</p>
